src/canister/individual_user_template/src/api/post/add_post_v2.rs

```diff
@@ -42,9 +42,11 @@ fn add_post_v2(post_details: PostDetailsFromFrontend) -> Result<u64, String> {
         )
     });
 
-    let post_id = response;
+    let post_id = response.clone().unwrap();
 
-    update_scores_and_share_with_post_cache_if_difference_beyond_threshold(&post_id);
+    if response.is_ok() {
+        update_scores_and_share_with_post_cache_if_difference_beyond_threshold(&post_id);
+    }
 
     if post_details.creator_consent_for_inclusion_in_hot_or_not {
         // * schedule hot_or_not outcome tabulation for the 48 hours after the post is created
```

Comment:
should this also be in the above if condition if is_ok

Reply:
Yes will remove it. Got added by clippy